tendrl/notifier/notification/__init__.py

```diff
@@ -73,26 +75,17 @@ def load_plugins(self):
             )
             raise ex
 
-    def save_alertnotificationconfig(self):
-        notification_config = {}
-        for n_plugin in NS.notification_medium:
-            for alert_type in NS.alert_types:
-                conf_name = '%s_%s' % (n_plugin, alert_type)
-                notification_config[conf_name] = "true"
-        NotificationConfig(config=notification_config).save()
-
     def __init__(self):
         super(NotificationPluginManager, self).__init__()
         try:
             self.load_plugins()
             notification_medium = []
+            self.complete = gevent.event.Event()
             for plugin in NotificationPlugin.plugins:
                 notification_medium.append(plugin.name)
-            NS.notification_medium = notification_medium
```

Comment:
Why are you removing this?
How would a user or even tendrl know the list of supported notification medium.
May not be a critical requirement but I feel it is good to have